google_attachment/static/src/js/AttachmentBox.js

```diff
@@ -68,10 +99,12 @@ function addImportFromGoogleToSideBar(clientId, apiKey){
         _createAttachmentFromGoogleDriveDoc(doc){
             var attachmentValues = {
                 name: doc.name,
+                datas_fname: buildFileNameWithExtension(doc.name, doc.mimetype),
+                mimetype: doc.mimetype,
                 type: "url",
                 url: doc.url,
-                res_id: this.env.activeIds[0],
-                res_model: this.env.model,
+                res_id: this.currentResID,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'res_id' is not in camel case. (camelcase)](https://app.codacy.com/app/numigi/odoo-web-addons/pullRequest?prid=3183161)